lib/main.js

```diff
@@ -43,8 +43,12 @@ class Lambda {
     this._createSampleFile('deploy.env', 'deploy.env')
     this._createSampleFile(program.contextFile, 'context.json')
     this._createSampleFile('event_sources.json', 'event_sources.json')
-    console.log('Setup done. Edit the .env, deploy.env, ' + program.contextFile + ' and ' + program.eventFile +
-      ' files as needed.')
+    console.log(
+      'Setup done. ' +
+      'Edit the .env, deploy.env, %s, event_sources.json and %s files as needed.',
+      program.contextFile,
+      program.eventFile
+    )
```

Comment:
might look nicer as:
```
console.log(`Setup done.
                       Edit the .env, deploy.env, ${program.contextFile}, event_sources.json and ${program.eventFile} as needed.`)
```

Reply:
OK!
(Because the width was long, I tried to adopt format to do some line breaks.)